src/aura/BGE_ConfigurationWizard/BGE_ConfigurationWizardController.js

```diff
@@ -1,118 +1,62 @@
 ({
```

Comment:
Wow. The controller is so readable!

Reply:
Right?! I think we learned a lot from doing it the other way, but am SO glad we've reworked this.